### DISCUSSION TITLE
v4.1.3

### DISCUSSION THREAD

**Discussion by awaterma** [salesforce/tough-cookie#285] [Announcements]
Security fix for Prototype Pollution discovery in #282. This is a minor release, although output from the `inspect` utility is affected by this change, we felt this change was important enough to be pushed into the next patch.

<hr /><em>This discussion was created from the release <a href='https://github.com/salesforce/tough-cookie/releases/tag/'>v4.1.3</a>.</em>

**Comment by BeigeBox** [comment]
Could you put together a list of which versions can directly upgrade to v4.1.3? There's a lot of projects which use this as a dep, and knowing which versions can update to v4.1.3 without likely regressions would be very helpful.

**Reply by Farouk1511** [reply]
Was any list published regarding this?
